### ISSUE TITLE
Focus does not move to the days of the week, and Talkback does not read the days of the week

### ISSUE THREAD

**Issue by rezeaka** [prolificinteractive/material-calendarview#834]
**Samsung Galaxy Note 5, Talkback version 5.2.1, also reported on a Tablet device**

- [ ] When swiping to the right on the calendar, after the "next" calendar button is read, focus moves to the entire calendar, Talkback reads, "Calendar", then with another swipe to the right, focus moves to the first date of the calendar ("1") and skips reading the days of the week,

- [ ] When the calendar opens and no focus is applied, Talkback reads the entire calendar content (including "S, M, T, W, T, F, S", but it is skipping the days of the week when focus is applied to any other  elements of the calendar
